spec/functions/simplib/bracketize_spec.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby -S rspec
+require 'spec_helper'
+
+describe 'simplib::bracketize' do
+  context 'should return converted values when conversion is possible' do
+    it 'converts an Array of networks' do
+      input = [
+        '2001:0db8:85a3:0000:0000:8a2e:0370:7334',
+        '2001:0db8:85a3:0000:0000:8a2e:0370:7334/24'
+      ]
+      expected_output = [
+        '[2001:0db8:85a3:0000:0000:8a2e:0370:7334]',
+        '[2001:0db8:85a3:0000:0000:8a2e:0370:7334]/24'
+      ]
+
+      is_expected.to run.with_params(input).and_return(expected_output)
+    end
+
+    it 'returns same input if not an ipv6' do
```

Comment:
really is a test that returns input untouched if no transformation is required.  One of the entries is an IPv6 address that already has brackets